python/mleap/sklearn/logistic.py

```diff
@@ -102,6 +110,8 @@ def deserialize_from_bundle(self, transformer, node_path, node_name):
             transformer.fit_intercept = False
 
         transformer.coef_ = np.array([transformer.coef_])
```

Comment:
just double checking, does this need to happen for both the binary and the multinomial logistic regression? or could it be the else statement in the if below?

Reply:
I turned this into an `np.reshape` operation to make the shape of `transformer.coef_` more explicit for binary classification